service/storage_service.cc

```diff
@@ -1116,8 +1116,7 @@ class topology_coordinator {
 
             topology_mutation_builder builder(guard.write_timestamp());
             builder.set_transition_state(topology::transition_state::commit_cdc_generation)
-                   .set_new_cdc_generation_data_uuid(gen_uuid)
-                   .del_global_topology_request();
+                   .set_new_cdc_generation_data_uuid(gen_uuid);
             auto reason = ::format(
```

Comment:
Please add a comment: we don't delete the request now, but only after the generation is committed. (And explain why.)

Reply:
Added.